test/integration/annotation_spec.js

```diff
@@ -87,3 +87,38 @@ describe("Checkbox annotation", () => {
     });
   });
 });
+
+describe("Text widget", () => {
+  describe("issue13271.pdf", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait("issue13271.pdf", "[data-annotation-id='24R']");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must all the fields with the same value", async () => {
```

Comment:
This reads a little bit weird, how about "all fields must have the same value" instead?